packages/@adobe/spectrum-css-temp/components/dropdown/index.css

```diff
@@ -157,16 +157,17 @@ governing permissions and limitations under the License.
 /* When used with a label or inside a Form, we need to override some things from .spectrum-Field
  * so quiet dropdowns still collapse properly. */
 .spectrum-Field.spectrum-Dropdown-fieldWrapper--quiet {
+  max-width: max-content;
+  inline-size: 100%;
```

Comment:
Allows the picker to grow to fit the container, but `max-width` stops it taking up unneeded space